package.json

```diff
@@ -8,6 +8,7 @@
   },
   "description": "OpenApi Specification Diff tool",
   "license": "MIT",
+  "type": "module",
```

Comment:
This caused this bug:

https://devdiv.visualstudio.com/DevDiv/_git/openapi-alps/pullrequest/559801

Reply:
Sorry I missed this in the review.  I should have at least asked why you added this.